v2/snmp/manager.go

```diff
@@ -2,19 +2,25 @@ package snmp
 
 import (
 	"context"
+	"encoding/asn1"
+	"fmt"
 	"net"
+	"strconv"
+	"strings"
+
+	"github.com/geoffgarside/ber"
 )
 
 // Manager provides an interface for SNMP device management.
 type Manager interface {
```

Comment:
Just a reminder to discuss this interface name further.

Reply:
What if we just called it Agent?  so agent.go, snmp.Agent.GetNext, etc.

Reply:
It's definitely NOT an agent.

Reply:
Sorry ignore, head up my a this morning